cmd/notation-azure-kv/main.go

```diff
@@ -34,3 +42,22 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func help() {
+	fmt.Printf(`notation-azure-kv - Notation - Notary V2 Azure KV plugin
+
+Usage:
+  notation-azure-kv <command>
+
+Version:
+  %s
+
+Commands:
+  get-plugin-metadata  Get plugin metadata
+  generate-signature   Sign artifacts with keys in Azure Key Vault
+  describe-key         Azure key description
+
+Documentation:
+  https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#plugin-contract
```

Comment:
We need to have a tagged version instead of `main`.

Reply:
Updated to use rc.2